
 The "-b" option allows to manually define size of sector size. We need to support
 4096 bytes for really huge disks. [kzak 12/14/2006]
 
--- util-linux-2.13-pre7/fdisk/fdisk.c.kzak	2006-12-14 10:21:57.000000000 +0100
+++ util-linux-2.13-pre7/fdisk/fdisk.c	2006-12-14 10:22:21.000000000 +0100
@@ -2491,7 +2491,7 @@
 			*/
 			sector_size = atoi(optarg);
 			if (sector_size != 512 && sector_size != 1024 &&
-			    sector_size != 2048)
+			    sector_size != 2048 && sector_size != 4096)
 				fatal(usage);
 			sector_offset = 2;
 			user_set_sector_size = 1;
